lib/whitehall_importer/create_migration.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "gds_api/whitehall_export"
+
+module WhitehallImporter
+  class CreateMigration
+    attr_reader :organisation_content_id, :document_type
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def initialize(organisation_content_id, document_type)
+      @organisation_content_id = organisation_content_id
+      @document_type = document_type
+    end
```

Comment:
I'm not sure quite on the status but we'll probably have to rebase this against Bruce's change that adds the sub types.

Reply:
Indeed 😬

Reply:
This has been rebased now 👍